gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -306,7 +306,7 @@ TFLiteCore::invoke (const GstTensorMemory * input, GstTensorMemory * output)
     for (int j = 0; j < num_of_input[i]; j++) {
       if (inputTensorMeta.info[i].type == _NNS_FLOAT32) {
         (interpreter->typed_tensor < float >(in_tensor))[j] =
-            (((float *) input[i].data)[j] - 127.5f) / 127.5f;
+            ((float *) input[i].data)[j];
```

Comment:
LGTM.

However, the code says you are doing memcpy. After completing multi-tensor issues, please start work on removing memcpy from tensorflowlite subplugin.